pkg/quota_management/billing_model.go

```diff
@@ -0,0 +1,10 @@
+package quota_management
+
+type BillingModel struct {
```

Comment:
Is this a KafkaBillingModel or an AMSBillingModel?

I think we should be more specific as it is not clear from just the name.

Reply:
I understand :+1: . What would be the difference between the QuotaListBillingModel and the KafkaBillingModel?

Reply:
`KafkaBillingModel` does not contains the values of 'how much quota' is assigned. The `QuotaListBillingModel` includes the values of how much 'is allowed' for that specific 'BillingModel' instance.

In other words, `KafkaBillingModel` is the **definition** of the list of billing models supported by KFM for each instance. At the same time, `QuotaBillingModel` is the **configuration** of the billing models for each instance type in the QUOTA-LIST manager.

Reply:
But even if `QuotaBillingModel` is the configuration of the billing models, the billing models themselves are kafka billing models right?  the configuration only accepts the values that exist for kafka billing models